src/pages/careers/locations/roadtown/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import { roadtown } from '../../_model/_locations/_locations'
+import { NewLocationLayout } from '../_location-layout-new-offices'
+import { SEO } from 'components/containers'
+import Layout from 'components/layout/layout'
+import { localize, WithIntl } from 'components/localization'
+
+const query = graphql`
+    query {
+        roadtown: file(relativePath: { eq: "careers/roadtown.jpg" }) {
+            childImageSharp {
+                gatsbyImageData(
+                    formats: [AUTO, WEBP]
+                    layout: CONSTRAINED
+                    quality: 100
+                    placeholder: NONE
+                )
+            }
+        }
+        living_in_roadtown: file(relativePath: { eq: "careers/living_in_roadtown.jpg" }) {
+            ...fadeIn
+        }
+        bvi_grid_3: file(relativePath: { eq: "careers/bvi_grid_3.jpg" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const Roadtown = () => {
+    const images = useStaticQuery(query)
+
+    return (
+        <Layout type="careers">
+            {/* ---------WILL FILL THIS SECTION ONCE SEO IS DONE------
```

Comment:
Don't keep this SEO part commented out, Please speak with Swati if needed on this